src/core/server/metrics/metrics_service.ts

```diff
@@ -26,6 +29,7 @@ export class MetricsService
 {
   private readonly logger: Logger;
   private readonly opsMetricsLogger: Logger;
+  private readonly analyticsMetricsLogger: Logger;
```

Comment:
The specific analytics logger serves as a way to monitor the same metrics as we ingest. However, the logger is set to `info` mode, which might be too generic.
We could change that to `debug` or if someone feels strongly that it's not needed, remove it.